src/packages/popover/popover.taro.tsx

```
@@ -230,11 +224,10 @@ export const Popover: FunctionComponent<
       }
     }
 
-    if (elWidth === 0) {
-      styles.visibility = 'hidden'
-    } else {
-      styles.visibility = 'initial'
-    }
+    elWidth === 0
+      ? (styles.visibility = 'hidden')
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)